src/knockout-postbox.js

```diff
@@ -139,7 +139,17 @@
 
         return this;
     };
-
+    //dispose all subscriptions by topic name.
+    exports.disposeTopic = function (topic) {
+        var sub = subscriptions[topic];
```

Comment:
@rniemeyer Arent subscriptions accessed using the subscription id instead of the topic ?

Compare https://github.com/rniemeyer/knockout-postbox/blob/master/src/knockout-postbox.js#L53 with this line...

Sorry, I am currently not in the situation to test this so I can be totally incorrect. In that case, just ignore my comment.